dbee/models/result.go

```diff
@@ -34,6 +34,8 @@ type (
 		Header() (Header, error)
 		Next() (Row, error)
 		Close()
+		SetCallback(callback func())
+		SetCustomHeader(header Header)
```

Comment:
I don't want this here, because these two methods aren't required to make a functioning `IterResult`

They are just helper methods in the `common.Result` builder for creating specific implementations of this interface.

Reply:
[b9d0035](https://github.com/kndndrj/nvim-dbee/pull/34/commits/b9d0035a1eeda0d6b564928c154cd39e1d7ea90e)